### PR TITLE
Adop 560 eligibility content update

### PR DESCRIPTION
**Pull request by dilipsamrahmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

https://tools.hmcts.net/jira/browse/ADOP-560
https://tools.hmcts.net/jira/browse/ADOP-554

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/steps/eligibility/domicile/content.test.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  beforeEach(() => {
+    generatedContent = generateContent(commonContent);
+  });
+
+  test('should return correct english content', () => {
+    expect(generatedContent.section).toEqual(enContent.section);
+    expect(generatedContent.label).toEqual(enContent.label);
+    expect(generatedContent.domicileNo).toEqual(enContent.domicileNo);
+    expect(generatedContent.moreInfo).toEqual(enContent.moreInfo);
+    expect(generatedContent.errors).toEqual(enContent.errors);
```
Comment (abidfs) [inline src/main/steps/eligibility/domicile/content.test.ts:53]: can be simplified using languageAssertions
example https://github.com/hmcts/adoption-web/blob/05ddc4ab44fa2e4bee1632469dc6df594ee7ca22/src/main/steps/sibling/placement-order-check-your-answers/content.test.ts#L51

File: src/main/steps/eligibility/domicile/content.test.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  });
+
+  test('should return correct welsh content', () => {
+    generatedContent = generateContent({ ...commonContent, language: 'cy' });
+    expect(generatedContent.section).toEqual(cyContent.section);
+    expect(generatedContent.label).toEqual(cyContent.label);
+    expect(generatedContent.domicileNo).toEqual(cyContent.domicileNo);
+    expect(generatedContent.moreInfo).toEqual(cyContent.moreInfo);
+    expect(generatedContent.errors).toEqual(cyContent.errors);
```
Comment (abidfs) [inline src/main/steps/eligibility/domicile/content.test.ts:62]: https://github.com/hmcts/adoption-web/blob/05ddc4ab44fa2e4bee1632469dc6df594ee7ca22/src/main/steps/sibling/placement-order-check-your-answers/content.test.ts#L51

File: src/main/steps/eligibility/domicile/content.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      label: l => l.label,
+      section: l => l.section,
+      labelHidden: false,
+      values: [
+        { label: l => l.yes, value: YesOrNo.YES },
+        {
+          label: l => l.no,
+          value: YesOrNo.NO,
+          conditionalText: l =>
+            `<p class="govuk-label">${l.domicileNo}</p> <p class ="govuk-label"><a href="https://www.gov.uk/child-adoption">${l.moreInfo}</a></p>`,
```
Comment (abidfs) [inline src/main/steps/eligibility/domicile/content.ts:47]:
```suggestion
            `<p class="govuk-label">${l.domicileNo}</p> <p class="govuk-label"><a href="https://www.gov.uk/child-adoption">${l.moreInfo}</a></p>`,
```

File: src/main/steps/eligibility/lived-uk/content.test.ts [hunk 4]
```diff
@@ -1,68 +1,85 @@
...
-      'You cannot apply to adopt a child unless you have a permanent home here.'
-    );
-    expect(generatedContent.one).toEqual('Yes');
-    expect(generatedContent.two).toEqual('No');
-    expect(generatedContent.hint).toEqual('You can answer yes if your permanent home is here.');
+    expect(generatedContent.section).toEqual(enContent.section);
+    expect(generatedContent.label).toEqual(enContent.label);
+    expect(generatedContent.livedUKNo).toEqual(enContent.livedUKNo);
+    expect(generatedContent.moreInfo).toEqual(enContent.moreInfo);
+    expect(generatedContent.errors).toEqual(enContent.errors);
```
Comment (abidfs) [inline src/main/steps/eligibility/lived-uk/content.test.ts:53]: whilst you are here, can you simplify this using https://github.com/hmcts/adoption-web/blob/05ddc4ab44fa2e4bee1632469dc6df594ee7ca22/src/main/steps/sibling/placement-order-check-your-answers/content.test.ts#L51

File: src/main/steps/eligibility/lived-uk/content.ts [hunk 5]
```diff
@@ -40,11 +37,15 @@ export const form: FormContent = {
...
       labelHidden: false,
       values: [
-        { label: l => l.one, value: YesOrNo.YES },
-        { label: l => l.two, value: YesOrNo.NO, conditionalText: l => `<p class="govuk-label">${l.livedUKNo}</p>` },
+        { label: l => l.yes, value: YesOrNo.YES },
+        {
+          label: l => l.no,
+          value: YesOrNo.NO,
+          conditionalText: l =>
+            `<p class="govuk-label">${l.livedUKNo}</p> <p class ="govuk-label"><a href="https://www.gov.uk/child-adoption">${l.moreInfo}</a></p>`,
```
Comment (abidfs) [inline src/main/steps/eligibility/lived-uk/content.ts:47]:
```suggestion
            `<p class="govuk-label">${l.livedUKNo}</p> <p class="govuk-label"><a href="https://www.gov.uk/child-adoption">${l.moreInfo}</a></p>`,
```

File: src/main/steps/eligibility/under-18/content.test.ts [hunk 6]
```diff
@@ -1,64 +1,89 @@
...
-    expect(generatedContent.under18No).toEqual(
-      'You can only apply to adopt a child if they are under 18 years old on the date your application is submitted.'
-    );
-    expect(generatedContent.one).toEqual('Yes');
-    expect(generatedContent.two).toEqual('No');
+    expect(generatedContent.section).toEqual(enContent.section);
+    expect(generatedContent.label).toEqual(enContent.label);
+    expect(generatedContent.under18No).toEqual(enContent.under18No);
+    expect(generatedContent.moreInfo).toEqual(enContent.moreInfo);
+    expect(generatedContent.errors).toEqual(enContent.errors);
```
Comment (abidfs) [inline src/main/steps/eligibility/under-18/content.test.ts:57]: same as above comments

File: src/main/steps/eligibility/under-18/content.test.ts [hunk 7]
```diff
@@ -1,64 +1,89 @@
...
-      'You can only apply to adopt a child if they are under 18 years old on the date your application is submitted. (in welsh)'
-    );
-    expect(generatedContent.one).toEqual('Yes (in welsh)');
-    expect(generatedContent.two).toEqual('No (in welsh)');
+    generatedContent = generateContent({ ...commonContent, language: 'cy' });
+    expect(generatedContent.section).toEqual(cyContent.section);
+    expect(generatedContent.label).toEqual(cyContent.label);
+    expect(generatedContent.under18No).toEqual(cyContent.under18No);
+    expect(generatedContent.moreInfo).toEqual(cyContent.moreInfo);
+    expect(generatedContent.errors).toEqual(cyContent.errors);
```
Comment (abidfs) [inline src/main/steps/eligibility/under-18/content.test.ts:66]: same as above comments

File: src/main/steps/eligibility/under-18/content.ts [hunk 8]
```diff
@@ -40,8 +39,13 @@ export const form: FormContent = {
...
       labelHidden: false,
       values: [
-        { label: l => l.one, value: YesOrNo.YES },
-        { label: l => l.two, value: YesOrNo.NO, conditionalText: l => `<p class="govuk-label">${l.under18No}</p>` },
+        { label: l => l.yes, value: YesOrNo.YES },
+        {
+          label: l => l.no,
+          value: YesOrNo.NO,
+          conditionalText: l =>
+            `<p class="govuk-label">${l.under18No}</p> <p class ="govuk-label"><a href="https://www.gov.uk/child-adoption">${l.moreInfo}</a></p>`,
```
Comment (abidfs) [inline src/main/steps/eligibility/under-18/content.ts:47]:
```suggestion
            `<p class="govuk-label">${l.under18No}</p> <p class="govuk-label"><a href="https://www.gov.uk/child-adoption">${l.moreInfo}</a></p>`,
```

File: src/main/steps/eligibility/under-21/content.test.ts [hunk 9]
```diff
@@ -1,63 +1,83 @@
...
-    expect(generatedContent.under21Yes).toEqual(
-      'You must be 21 or over to adopt a child. This includes any other applicant.'
-    );
-    expect(generatedContent.one).toEqual('Yes');
-    expect(generatedContent.two).toEqual('No');
+    expect(generatedContent.section).toEqual(enContent.section);
+    expect(generatedContent.label).toEqual(enContent.label);
+    expect(generatedContent.under21Yes).toEqual(enContent.under21Yes);
+    expect(generatedContent.moreInfo).toEqual(enContent.moreInfo);
+    expect(generatedContent.errors).toEqual(enContent.errors);
```
Comment (abidfs) [inline src/main/steps/eligibility/under-21/content.test.ts:51]: same as above

File: src/main/steps/eligibility/under-21/content.test.ts [hunk 10]
```diff
@@ -1,63 +1,83 @@
...
-      'You must be 21 or over to adopt a child. This includes any other applicant. (in welsh)'
-    );
-    expect(generatedContent.one).toEqual('Yes (in welsh)');
-    expect(generatedContent.two).toEqual('No (in welsh)');
+    generatedContent = generateContent({ ...commonContent, language: 'cy' });
+    expect(generatedContent.section).toEqual(cyContent.section);
+    expect(generatedContent.label).toEqual(cyContent.label);
+    expect(generatedContent.under21Yes).toEqual(cyContent.under21Yes);
+    expect(generatedContent.moreInfo).toEqual(cyContent.moreInfo);
+    expect(generatedContent.errors).toEqual(cyContent.errors);
```
Comment (abidfs) [inline src/main/steps/eligibility/under-21/content.test.ts:60]: same as above

File: src/main/steps/eligibility/under-21/content.ts [hunk 11]
```diff
@@ -38,8 +37,13 @@ export const form: FormContent = {
...
       labelHidden: false,
       values: [
-        { label: l => l.one, value: YesOrNo.YES },
-        { label: l => l.two, value: YesOrNo.NO, conditionalText: l => `<p class="govuk-label">${l.under21Yes}</p>` },
+        { label: l => l.yes, value: YesOrNo.YES },
+        {
+          label: l => l.no,
+          value: YesOrNo.NO,
+          conditionalText: l =>
+            `<p class="govuk-label">${l.under21Yes}</p> <p class ="govuk-label"><a href="https://www.gov.uk/child-adoption">${l.moreInfo}</a></p>`,
```
Comment (abidfs) [inline src/main/steps/eligibility/under-21/content.ts:45]:
```suggestion
            `<p class="govuk-label">${l.under21Yes}</p> <p class="govuk-label"><a href="https://www.gov.uk/child-adoption">${l.moreInfo}</a></p>`,
```

**Review by abidfs — CHANGES_REQUESTED** [review]
need to fix few typos

**Review by abidfs — APPROVED** [review]

**Review by pankajhmcts — APPROVED** [review]

**Comment by pankajhmcts** [comment]
@dilipsamrahmcts @nraghu32 @gauravtomar-gh @abidfs 
Merging this PR after testing complete confirmation received.

**Pull request merged**